openslides_backend/action/actions/mediafile/move.py

```diff
@@ -78,10 +78,11 @@ def prepare_move_data(
                         f"Moving item {id_} to one of its children is not possible."
                     )
         for id_ in ids:
-            if id_ not in db_instances or db_instances[id_].get("owner_id") != owner_id:
-                raise ActionException(f"Id {id_} not in db_instances.")
-
-            instance: Dict[str, Any] = {"id": id_, "parent_id": parent_id}
+            instance: Dict[str, Any] = {
+                "id": id_,
+                "parent_id": parent_id,
+                "owner_id": owner_id,
+            }
```

Comment:
Why did you change this? Is it now checked anywhere that no additional instances (which are not in the db) can be given?

Reply:
The check permission code need the owner id , so it can check the parent/owner_id vs id/owner_id.

Reply:
Ok, but you removed the check that not additional ids which are not in the datastore can be given. Is this checked anywhere else? If not, add it back (and also add a test for this, since seemingly it did not trigger an error anywhere).

Reply:
Ahh, yes, I add the db_instance check again, the owner_id check is done somewhere else.